pkg/policy/l4.go

```diff
@@ -273,26 +304,31 @@ func ParsePortProtocol(port int, protocol string) (np NamedPort, err error) {
 	if port < 1 || port > 65535 {
 		return np, fmt.Errorf("Port number %d out of 16-bit range", port)
 	}
-	return NamedPort{
+	return PortProto{
 		Proto: uint8(u8p),
 		Port:  uint16(port),
 	}, nil
 }
 
-func (npm NamedPortsMap) AddPort(name string, port int, protocol string) error {
+func (npm NamedPortMap) AddPort(name string, port int, protocol string) error {
 	name, err := ValidatePortName(name)
 	if err != nil {
 		return err
 	}
-	np, err := ParsePortProtocol(port, protocol)
+	pp, err := ParsePortProto(port, protocol)
 	if err != nil {
 		return err
 	}
-	npm[name] = np
+	npm[name] = pp
 	return nil
 }
 
-func (npm NamedPortsMap) GetNamedPort(name string, proto uint8) (port uint16, err error) {
+// NamedPortMap abstracts different maps that implement GetNamedPort method
+type NamedPortsMap interface {
```

Comment:
nit: The comment is not consistent with the name of the interface.

Also, perhaps the `NamedPortMap` implementation should be something like `NamedPortSingleMap` to make it easier to tell the difference & not confuse whether code is referring to the interface or the concrete implementation?

Reply:
This interface is now removed in the latest version. Naming changes will have to wait for a folloow-on PR, if still deemed necessary.